lib/add_movie/domain/movie.dart

```diff
@@ -0,0 +1,28 @@
+class Movie {
```

Comment:
you already have Movie class inside lib/movie/domain/movie.dart
pls remove this class and use already existing class